src/components/atoms/faq/FaqItem.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FaqItemContainer, Answer} from './style';
+
+
+
+const FaqItem = ({ faq, isActive, onClick }) => {
+  return (
+    <FaqItemContainer className={isActive ? 'active' : ''} onClick={onClick}>
```

Comment:
isActive state should be managed in each component. It would be easier for you also.

Reply:
@Shurtu-gal  I have done this may be , check my this recent commit,
I will also delete the comments once I get the confirmation.
pls review this